compiler-rt/test/lit.common.configured.in

```diff
@@ -51,6 +51,7 @@ set_default("expensive_checks", @LLVM_ENABLE_EXPENSIVE_CHECKS_PYBOOL@)
 set_default("test_standalone_build_libs", @COMPILER_RT_TEST_STANDALONE_BUILD_LIBS_PYBOOL@)
 set_default("has_compiler_rt_libatomic", @COMPILER_RT_BUILD_STANDALONE_LIBATOMIC_PYBOOL@)
 set_default("aarch64_sme", @COMPILER_RT_HAS_AARCH64_SME_PYBOOL@)
+set_default("host_link_version", "@HOST_LINK_VERSION@")
```

Comment:
- [ ]  Many of the other vars here have a `COMPILER_RT` prefix.  Should we do the same or does it imply some additional semantics ("user option" in CMAKE script)?
- [ ] `set_default("darwin_linker_version", "@COMPILER_RT_DARWIN_LINKER_VERSION@")`